.github/workflows/deploy_new_account.yml

```diff
@@ -21,12 +21,22 @@ env:
   POLYGON_MAIN_NETWORK_FLAG: "0b0001_0001"
   POLYGON_MAIN_CHAIN_ID: "137"
 
-  PRIVADO_MAIN_CONTRACT_ADDRESS: "0x975556428F077dB5877Ea2474D783D6C69233742"
+  POLYGON_ZKVM_CONTRACT_ADDRESS: "0x3C9acB2205Aa72A05F6D77d708b5Cf85FCa3a896"
+  POLYGON_ZKVM_SENDER_DID: "did:iden3:polygon:zkevm:xJa94Ky3oeawnDdQVoTotncYnBA1fCLW6C4NZVpRu"
+  POLYGON_ZKVM_NETWORK_FLAG: "0b0001_0100"
+  POLYGON_ZKVM_CHAIN_ID: "1101"
+
+  POLYGON_CARDONA_CONTRACT_ADDRESS: "0x3C9acB2205Aa72A05F6D77d708b5Cf85FCa3a896"
```

Comment:
Better rename to POLYGON_ZKEVM_CARDONA.
Btw, where are these env vars are used?

Reply:
yeap, I forgot to add to yaml file